yucca/network_architectures/networks/YuccaNet.py

```diff
@@ -57,7 +59,15 @@ def predict(self, mode, data, patch_size, overlap, mirror=False):
             pred /= div
         return pred
 
-    def _predict3D(self, data, patch_size, overlap):
+    def _predict(self, data):
```

Comment:
`YuccaNet` now contains two methods `_predict` and `predict` but the two methods are quite different. I would prefer if one of the two methods had names which made it clear how they are different. My guess would be to rename `_predict` to something like `_full_size_predict` or `_standard_predict` or similar.